flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointProperties.java

```diff
@@ -89,6 +94,11 @@ boolean forceCheckpoint() {
         return forced;
     }
 
+    /** Returns whether the checkpoint has been restored in a {@link RestoreMode#NO_CLAIM} mode. */
```

Comment:
```suggestion
    /** Returns whether the checkpoint should be restored in a {@link RestoreMode#NO_CLAIM} mode. */
```

Reply:
Both versions are actually correct.

Reply:
I still prefer to use `should be`, which follows other comments.